core/ipc/rpc/utils/src/main/java/org/opennms/core/rpc/utils/mate/EntityScopeProvider.java

```diff
@@ -31,9 +31,15 @@
 import java.net.InetAddress;
 
 public interface EntityScopeProvider {
-    Scope getScopeForNode(final Integer nodeId);
 
-    Scope getScopeForAssets(final Integer nodeId);
+    interface ScopeId {
```

Comment:
To be more clear, it should be called `Contexts`. The terminology here is that the meta-data consists of (context, key, value) triplets whereas the (context, key) tuple is used to access a value. The scopes are just used be more flexible during interpolation.

Reply:
fixed